TLM/TLM/GlobalSuppressions.cs

```diff
@@ -0,0 +1,9 @@
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1117:Parameters should be on same line or separate lines", Justification = "<Pending>", Scope = "member", Target = "~M:TrafficManager.Manager.Impl.RoutingManager.RecalculateLaneEndRoutingData(System.UInt16,System.Int32,System.UInt32,System.Boolean)")]
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1117:Parameters should be on same line or separate lines", Justification = "<Pending>", Scope = "member", Target = "~M:TrafficManager.Manager.Impl.RoutingManager.ShouldReverseSimilarLaneIndex(System.Boolean,NetInfo.Lane,System.Boolean,System.Boolean,NetInfo.Lane,System.Boolean)~System.Boolean")]
```

Comment:
Can't we just add these suppressions in the file itself?

Reply:
It would be bulky.

Reply:
I will do this for the sake of consistency with the rest of TMPE